classroom.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'student'
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @students << student
+    student.classroom = self
```

Comment:
great for implementing the `has-many`/`belongs-to` relationship between student and classroom :+1: